### ISSUE TITLE
CP2077 crashing after a few seconds RTX3080 Win 11

### ISSUE THREAD

**Issue by MrUnderhill323** [Nukem9/dlssg-to-fsr3#92]
Installation works perfectly, I load into the game and I'm getting 120fps at 1440p where before I was just scraping 60 - brilliant! But after 15-20 seconds the game crashes, or freezes my PC so I have to force a restart with CTRL/ALT/DEL. No HDR on in game, Vsync turned off. I am using auto HDR from Windows and a reshade preset, so I'm not sure if they're causing the crashes? Any help appreciated.

**Comment by Styphoryte** [comment]
Same issue unfortunately, I managed to get to the main menu. But after enabling Frame Gen and pressed continue it started to load and then instant crash to desktop. Then now when I open it, instant error message Cyberpunk has crashed. :c I managed to get this working fine with Alan Wake 2. Not sure what the issue is here, I do have a lot of mods installed along with Reshade but I have reshade working fine in Alan Wake 2 with dlssg to fsr3 as well so not sure.

**Comment by D3vast8r** [comment]
Yup same here a bit different for me im only used 1 re-shade hdr, and nothing else even when I turn off ray tracing or path tracing I play the game a little load in running around but when I get to more stressful areas such as japan town or freaking jig jig street forget it, it freezes and crash.

**Comment by Styphoryte** [comment]
> Yup same here a bit different for me im only used 1 re-shade hdr, and nothing else even when I turn off ray tracing or path tracing I play the game a little load in running around but when I get to more stressful areas such as japan town or freaking jig jig street forget it, it freezes and crash.

Okay interesting, mine just refused to even load I clicked Continue after enabling Frame gen with DLSS on Balanced and then it started to load few seconds and then I got a Crash error report. Only that that fixed it was manually going to "%LOCALAPPDATA%\CD Projekt Red\Cyberpunk 2077" and then opening UserSettings.json find "DLSS_D" make sure the value is set to false as well as false for the default value and then my game would launch...

I may try .41 version I've heard a few ppl say that works still so not sure. I'm on Win 11 23H2 as well, newest Nvidia driver, etc.
